### DISCUSSION TITLE
Open oh-my-bash by default macos

### DISCUSSION THREAD

**Discussion by ericjb20** [ohmybash/oh-my-bash#347] [Q&A]
Are there any instructions for how to get macos to open oh my bash by default after installation? As of now the only way it works for me is to open a terminal window and type 'bash'. Even though echo $SHELL displays "/bin/bash".

Trying to get this to work in visual studio code as a default with my bash terminals and thinking it needs to do it on the macos itself first.

Thanks for any help. As of now I don't see any clear instruction for setting it up.

Update:

I created a .bash_profile file in my home directory with the following content:

 if [ -f ~/.bashrc ]; then
  . ~/.bashrc
fi

Also looked to see if I had this file already which I didn't. Some suggest you may have a .profile file instead.. Didn't have that either. For now this works and gets my terminal to use oh-my-bash on both my macos and visual studio code

**Comment by akinomyoga** [comment]
I think #302 is the same issue. Some solution seems to be provided by https://github.com/ohmybash/oh-my-bash/issues/302#issuecomment-1172986783, but I haven't checked it because I don't have macOS.

> For now this works and gets my terminal to use oh-my-bash on both my macos and visual studio code

Does it mean you have solved the issue?

**Reply by ericjb20** [reply]
Yea this resolved it for me!
